--- Makefile.am	2022-05-19 05:56:01.473574895 -0400
+++ Makefile.am	2022-05-19 05:56:45.965807233 -0400
@@ -16,7 +16,9 @@
 systemddir = /lib/systemd/system
 systemd_DATA = ukui-media-control-mute-led.service
 
-gsettingsschema_in_files = org.ukui.media.gschema.xml.in
+gsettingsschema_in_files = org.ukui.media.gschema.xml.in \
+	org.ukui.sound.gschema.xml.in \
+	org.ukui.audio.gschema.xml.in
 gsettings_SCHEMAS = $(gsettingsschema_in_files:.xml.in=.xml)
 @GSETTINGS_RULES@
 @INTLTOOL_XML_RULE@
