--- src/plugins/styles/styles.pro	2022-05-17 01:45:50.646690193 -0900
+++ src/plugins/styles/styles.pro	2022-05-17 01:49:15.100030991 -0900
@@ -1,14 +1,2 @@
 TEMPLATE = subdirs
-SUBDIRS = \
-    cleanlooks \
-    motif \
-    plastique
-
-greaterThan(QT_MAJOR_VERSION, 5) | greaterThan(QT_MINOR_VERSION, 6) {
-    # only 5.7 or later
-    SUBDIRS += bb10style
-}
-
-packagesExist(gtk+-2.0 x11) {
-    SUBDIRS += gtk2
-}
+SUBDIRS = gtk2
